src/qibolab/instruments/zhinst.py

```diff
@@ -750,14 +788,14 @@ def measure_relax(self, exp, qubits, relaxation_time, acquisition_type):
         elif len(self.sequence_qibo.qd_pulses) != 0:
             play_after = self.play_after_set(self.sequence_qibo.qd_pulses, "drive")
 
-        for qubit in qubits.values():
-            if qubit.flux_coupler:
-                continue
-            q = qubit.name
-            if len(self.sequence[f"readout{q}"]) != 0:
-                for pulse in self.sequence[f"readout{q}"]:
-                    i = 0
-                    with exp.section(uid=f"sequence_measure{q}", play_after=play_after):
+        with exp.section(uid=f"sequence_measure", play_after=play_after):
```

Comment:
Can you leave a comment somewhere about why you inverted the context and the loop?

Reply:
It allows multiplex from #464, as it was a single line I put here as well.

Reply:
Perfectly fine. I was just asking to leave a comment in the code, just to avoid wondering in a few months whether we should invert them again :)